File: CS/DesignerSample/Form1.cs
using DevExpress.XtraCharts;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace DesignerSample
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            dashboardDesigner1.CreateRibbon();
            dashboardDesigner1.LoadDashboard(@"..\..\Data\T597206.xml");
        }

        private void dashboardDesigner1_DashboardItemControlCreated(object sender, DevExpress.DashboardWin.DashboardItemControlEventArgs e)
        {
            if (e.DashboardItemName == "chartDashboardItem1")
            {
                e.ChartControl.AxisScaleChanged += ChartControl_AxisScaleChanged;
            }
        }

        private void ChartControl_AxisScaleChanged(object sender, AxisScaleChangedEventArgs e)
        {
            if (e.Axis is SecondaryAxisY)
            {
                if (e.GridSpacingChange.NewValue<1)
                {
                    e.Axis.NumericScaleOptions.GridSpacing = 1;
                }
            }
        }

        private void dashboardDesigner1_DashboardItemBeforeControlDisposed(object sender, DevExpress.DashboardWin.DashboardItemControlEventArgs e)
        {
            if (e.DashboardItemName == "chartDashboardItem1")
            {
                e.ChartControl.AxisScaleChanged -= ChartControl_AxisScaleChanged;
            }
        }
    }
}

File: VB/DesignerSample/Form1.vb
Imports DevExpress.XtraCharts
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace DesignerSample
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			dashboardDesigner1.CreateRibbon()
			dashboardDesigner1.LoadDashboard("..\..\Data\T597206.xml")
		End Sub

		Private Sub dashboardDesigner1_DashboardItemControlCreated(ByVal sender As Object, ByVal e As DevExpress.DashboardWin.DashboardItemControlEventArgs) Handles dashboardDesigner1.DashboardItemControlCreated
			If e.DashboardItemName = "chartDashboardItem1" Then
				AddHandler e.ChartControl.AxisScaleChanged, AddressOf ChartControl_AxisScaleChanged
			End If
		End Sub

		Private Sub ChartControl_AxisScaleChanged(ByVal sender As Object, ByVal e As AxisScaleChangedEventArgs)
			If TypeOf e.Axis Is SecondaryAxisY Then
				If e.GridSpacingChange.NewValue<1 Then
					e.Axis.NumericScaleOptions.GridSpacing = 1
				End If
			End If
		End Sub

		Private Sub dashboardDesigner1_DashboardItemBeforeControlDisposed(ByVal sender As Object, ByVal e As DevExpress.DashboardWin.DashboardItemControlEventArgs) Handles dashboardDesigner1.DashboardItemBeforeControlDisposed
			If e.DashboardItemName = "chartDashboardItem1" Then
				RemoveHandler e.ChartControl.AxisScaleChanged, AddressOf ChartControl_AxisScaleChanged
			End If
		End Sub
	End Class
End Namespace
